docs/guides/index.md

```diff
@@ -7,10 +7,14 @@ tags:
     - how to
 ---
 
+!!! tip "Need help?"
+    Get your questions answered with a Prefect product advocate by [Booking A Rubber Duck](https://calendly.com/prefect-experts/prefect-product-advocates)!
```

Comment:
Can we change the link to:
https://calendly.com/prefect-experts/prefect-product-advocates?utm_campaign=prefect_docs_guide&utm_content=prefect_docs&utm_medium=docs&utm_source=dpcs